sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1076,6 +1076,12 @@ public partial class ServerCertificateCustomValidationArgs
 }
 namespace Azure.Core.Serialization
 {
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class)]
+    public sealed partial class DefaultSubClassAttribute : System.Attribute
```

Comment:
Subclass should be one word.  See https://learn.microsoft.com/en-us/dotnet/api/system.type.issubclassof?view=net-7.0#system-type-issubclassof(system-type) for prior art.  I would also call this `UnknownSubclassAttribute` because it's not really a default any customer should be using.